internal/saucecloud/espresso.go

```diff
@@ -33,6 +36,11 @@ type EspressoRunner struct {
 func (r *EspressoRunner) RunProject() (int, error) {
 	exitCode := 1
 
+	if r.DryRun {
+		r.dryRun(r.Project.Espresso.App, r.Project.Espresso.TestApp)
```

Comment:
The `error` from `dryRun()` is being ignored ⚠️

Reply:
fixed!

Reply:
No it's not ;)

Reply:
updated 🤔